management/dns_update.py

```diff
@@ -303,6 +303,23 @@ def has_rec(qname, rtype, prefix=None):
 		if not has_rec(qname, rtype):
 			records.append((qname, rtype, value, explanation))
 
+	# Adds autoconfiguration A records for all domains.
+	# mta-sts.* - required A record for mta-sts (serving the policy)
+	mta_sts_records = [
+		("mta-sts", "A", env["PUBLIC_IP"], "Provides MTA-STS support"),
+		("mta-sts", "AAAA", env.get('PUBLIC_IPV6'), "Provides MTA-STS support"),
+		("_mta-sts", "TXT", "v=STSv1; id="+datetime.datetime.now().strftime("%Y%m%d%H%M%S")+"Z", "Enables MTA-STS support")
+	]
+
+	# Skip if the user has set a custom _smtp._tls record.
+	if not has_rec("_smtp._tls", "TXT", prefix="v=TLSRPTv1;"):
+		mta_sts_records.append(("_smtp._tls",  "TXT", "v=TLSRPTv1;", "change to a custom record like 'v=TLSRPTv1; rua=mailto:email@address' for reporting"))
```

Comment:
Verbiage should be clarified on how to receive reports or maybe enabled by defaults.

Reply:
I had the idea that reports should be enabled when an email address tlsrpt@box.<domainname> is available. But these are my first python changes, so step by step;)

Reply:
I changed this in a new commit made it more verbose.

Reply:
I forget to close this item; a more verbose string is added.